### ISSUE TITLE
Concurrently Display Multiple heatmaps

### ISSUE THREAD

**Issue by birm** [camicroscope/caMicroscope#347]
Currently, only one heatmap is rendered at a time (not counting the side-by-side second image may have a second heatmap).

@nanli-emory can confirm/deny, but I believe that outside of possible performance hits, the concern is usability - it's difficult to display many different gradients at once without ambiguity.

So, this issue is mostly about resolving _that_ issue; what considerations/colors/UX would make it reasonable to view more than one heatmap at a time? How many can we display before the slide just looks like a mess? Should that be a hard limit?

One idea we've had before is to have up to three heatmaps, one each assigned to red, green, and blue gradients. This wouldn't change any saved heatmap display settings, rather override them in this specific case.

**Comment by cjchirag7** [comment]
@birm , does entirely different mean the heatmaps may be of two different slides also ?

**Comment by Hemansh31** [comment]
I don't think so, becoz forcing colours to behave as a single colour gradient means they are on the same slide. Well, that's what I think anyways.

**Comment by spursbyte** [comment]
I will like to work on this issue. can you give me some more info?

**Comment by birm** [comment]
Sorry for the delayed response.

We expect heatmaps to be on the same slide, at least for it to make any sense to display. Coordinated slide viewing is a different thing altogether. 

I'll update the description with hopefully more useful details, but feel free to ask any follow up questions you may have.

**Comment by spursbyte** [comment]
> Sorry for the delayed response.
> 
> We expect heatmaps to be on the same slide, at least for it to make any sense to display. Coordinated slide viewing is a different thing altogether.
> 
> I'll update the description with hopefully more useful details, but feel free to ask any follow up questions you may have.

thank you. I really look forward to contribute to your organistion .I will be highly grateful if you help me get started with any issue.
